src/edge/importer.py

```diff
@@ -94,23 +94,26 @@ def build_fragment(self):
             [f[0] for f in self.__features]
             + [f[1] + 1 for f in self.__features]))
         break_points = sorted(break_points)
+
+        cur_len = 0
         chunk_sizes = []
+        seqlen = len(self.__sequence)
```

Comment:
how about rename seqlen as seq_len? just to be consistent with cur_len. If you care, you can rename to be more readable names like sequence_length and current_length. I don't have too strong opinion on this since len is a common name in Python.